frontend/src/scenes/trends/personsModalLogic.ts

```diff
@@ -101,6 +102,10 @@ export const personsModalLogic = kea<personsModalLogicType<PersonModalParams>>({
             (s) => [s.peopleLoading, s.loadingMorePeople],
             (peopleLoading, loadingMorePeople) => peopleLoading && !loadingMorePeople,
         ],
+        clickhouseFeaturesEnabled: [
```

Comment:
This selector is duplicated from `funnelLogic`, was considering abstracting it, but as it's straightforward it's probably fine.

Reply:
Yeah, this should be hoisted upwards, perhaps just to preflightLogic.